sdk/search/test-resources.bicep

```diff
@@ -1,22 +1,38 @@
-param searchEndpointSuffix string = 'search.windows.net'
-param storageEndpointSuffix string = 'core.windows.net'
-param location string = '${resourceGroup().location}'
+param environmentEndPoints object = {
```

Comment:
We should pass in this endpoint from the test infrastructure rather than defining these endpoints here, and keep the original `searchEndpointSuffix string = 'search.windows.net` as that's just a default that will get overridden by the `New-TestResources.ps1` script.